frontend/pluginloader.lua

```diff
@@ -261,6 +265,23 @@ function PluginLoader:createPluginInstance(plugin, attr)
     end
 end
 
+--- Tries to stop and unload a specific plugin instance if it's currently running.
+function PluginLoader:stopPluginInstance(name)
+    if not name then return false, "no plugin name provided" end
+    local instance = self.loaded_plugins and self.loaded_plugins[name]
+    if not instance then return false, "not loaded" end
+
+    local ok, err = false, "no stop method"
+    local fn = instance.stop
```

Comment:
Maybe not the best name?

Reply:
They do it with a check:
https://github.com/koreader/koreader/blob/7e63f91c8e74af64089cefa187a17d664e261b35/plugins/SSH.koplugin/main.lua#L125-L126
https://github.com/koreader/koreader/blob/7e63f91c8e74af64089cefa187a17d664e261b35/plugins/httpinspector.koplugin/main.lua#L41-L42

Reply:
Yes, so maybe something like `stopPlugin`.